test/e2e/common/node/container_probe.go

```diff
@@ -25,12 +25,16 @@ import (
 	"strings"
 	"time"
 
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/watch"
```

Comment:
I'm surprised the linter or verify scripts don't complain for having this deps separated from the others 🤷